src/poprox_storage/repositories/accounts.py

```diff
@@ -145,3 +156,19 @@ def end_subscription_for_account(self, account_id: UUID):
             .values(ended=sqlalchemy.text("NOW()"))
         )
         self.conn.execute(delete_query)
+
+    def record_consent(self, account_id: UUID, document_name: str):
+        consent_tbl = self.tables["account_consent_log"]
+        query = sqlalchemy.insert(consent_tbl).values(
+            account_id=account_id, document_name=document_name
+        )
+        self.conn.execute(query)
+
+    def update_status(self, account_id: UUID, new_status: str):
```

Comment:
Would it make sense to do these two database operations as part of the same transaction?

Reply:
I think it would. I don't really know how transactions are best managed in this data design. If it's just a connection property you can easily make a transaction before building the repository object and then commit after right?